build.xml

```diff
@@ -357,7 +360,7 @@
   </exec>
 </target>
 
-<target name="dist.oneclick" depends="generated, timer-jar, docs-dist"
+<target name="dist.oneclick" if="is_windows" depends="generated, timer-jar, docs-dist"
```

Comment:
This target actually works on non-Windows platforms (as long as the Uniserver files are alongside).

Reply:
These would need to be manually installed alongside as line 327 does not extract them on non-windows os. 
<target name="unpack-uniserver" if="is_windows"

Is there a reason to build the Windows installer on Linux?

Reply:
I normally build all the distributable artifacts on MacOS.  You're correct that extracting the UniServer files has to be done on Windows, but then the files can be copied elsewhere.

Rather than conditionalize the target itself, I'd rather conditionalize the behavior to test for the presence of the UniServer sources, and present an obvious but non-fatal message if they're absent.

http://ant.apache.org/manual/Tasks/available.html